src/Console/Application.php

```diff
@@ -215,6 +221,24 @@ private function initializeApplication(InputInterface $input): TaskDescriptorCol
             ));
         }
 
+        $this->getDefinition()->addOption(
+            new InputOption(
+                'no-remote',
+                null,
+                InputOption::VALUE_NONE,
+                'Skips the import of all remote remote packages.',
```

Comment:
No trailing dot here

```suggestion
                'Skips the import of all remote remote packages',
```